Examples/MDFTestingExample/MDFTestingExampleTests/MDFTestingExampleTests.m

```diff
@@ -0,0 +1,37 @@
+//
+//  MDFTestingExampleTests.m
+//  MDFTestingExampleTests
+//
+//  Created by Wenyu Zhang on 9/24/19.
+//  Copyright © 2019 Wenyu Zhang. All rights reserved.
+//
```

Comment:
Please update these copyright headers with the apache licenses.

Reply:
Addressed in b5f238d2c4d4f1be6fe1881df69fc5b91ed1e6c2.